data/assets/scene/digitaluniverse/brown_dwarfs.asset

```diff
@@ -0,0 +1,69 @@
+local textures = asset.resource({
+  Name = "Brown Dwarf Textures",
+  Type = "HttpSynchronization",
+  Identifier = "digitaluniverse_brown_dwarfs_textures",
```

Comment:
```suggestion
  Identifier = "digitaluniverse_dwarfs_textures",
```
Without this, OpenSpace will not start with the given data files. It's the same as the one for white dwarfs. However, they currently use different names and this should be changed. 

Since it only really includes a point texture, maybe we should create a new identifier and synced dataset for just that point texture, instead of tying it to the specific dataset?

@alexanderbock need to discuss this